core/model/itinerary/itinerary.go

```diff
@@ -0,0 +1,67 @@
+package itinerary
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/openmarketplaceengine/openmarketplaceengine/core/model/job"
+	"github.com/openmarketplaceengine/openmarketplaceengine/core/model/step"
+)
+
+// Itinerary is a planned journey for a job.Job array.
+type Itinerary struct {
+	ID          string
+	Jobs        []*job.Job
+	Steps       []*step.Step
+	CurrentStep step.Step
```

Comment:
CurrentStep should be an `int` index for `[]Steps`

Reply:
yep, good catch